apps/nextjs-website/react-components/types/index.ts

```diff
@@ -17,5 +18,6 @@ export type {
   CardsProps,
   FooterProps,
   EditorialSwitchProps,
-  PreHeaderProps
+  PreHeaderProps,
+  StripeLinkProps
```

Comment:
```suggestion
  StripeLinkProps,
```
Don't know why lint isn't doing this automatically, but it will reduce lines changed for future PRs